README.md

```diff
@@ -1,2 +1,27 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Hello-Microverse
-Hello World
+
+> One paragraph statement about the project.
+
+![aaaaaa](https://user-images.githubusercontent.com/63915024/143270765-932b9ad8-e2e7-4365-a549-2ca45bd7ec04.png)
```

Comment:
- [ ] OPTIONAL very nice of you to add an image here. In general, though you should usually give the the text in the square brackets some significant meaning. this is like the `alt` attribute on an `<img />` tag.